### PR TITLE
Update bcdc_dataset.json

### PR DESCRIPTION
**Pull request by annikaLiving**
Changed the label and added Help text.

### CODE REVIEW

File: ckanext/bcgov_schema/bcdc_dataset.json [hunk 1]
```diff
@@ -92,8 +92,8 @@
     },
     {
       "field_name": "support_url",
-      "label": "Open a ticket for support",
-      "form_placeholder": "URL to a request service or help desk portal."
+      "label": "Submit a support request",
+      "help_text": "URLs to a support service or portal."
```
Comment (anoshashfaq) [inline ckanext/bcgov_schema/bcdc_dataset.json:96]:
```suggestion
      "help_text": "URL to a support service or portal."
```

File: ckanext/bcgov_schema/bcdc_dataset.json [hunk 2]
```diff
@@ -92,8 +92,8 @@
     },
     {
       "field_name": "support_url",
-      "label": "Open a ticket for support",
-      "form_placeholder": "URL to a request service or help desk portal."
+      "label": "Submit a support request",
```
Comment (anoshashfaq) [inline ckanext/bcgov_schema/bcdc_dataset.json:95]:
```suggestion
      "label": "Create a support request",
```

**Review by anoshashfaq — COMMENTED** [review]
Will the help text be displayed?

**Comment by annikaLiving** [comment]
> Will the help text be displayed?

Yes, I had done the form help text initially.

**Pull request merged**